.github/workflows/fix-format.yml

```diff
@@ -31,9 +31,9 @@ jobs:
       - name: Setup Node
         uses: actions/setup-node@49933ea5288caeca8642d1e84afbd3f7d6820020 # v4.4.0
         with:
-          node-version-file: './server/.nvmrc'
-          cache: 'npm'
-          cache-dependency-path: '**/package-lock.json'
+          node-version-file: './.github/.nvmrc'
```

Comment:
Isn't this still wrong?

Reply:
Looking more carefully, this is actually formatting all of the projects, not just .github. You'd technically have to run a version of setup-node for each project. This all may be a moot point, since the node version is always the same for every project. 

Would it make sense to just treat the node version as a workspace level setting, instead of per-project? Then, the nvmrc file could be moved up to the root directory of the git repo.